src/w3d/renderer/sortingrenderer.cpp

```diff
@@ -271,14 +271,17 @@ void SortingRendererClass::Insert_Triangles(const SphereClass &bounding_sphere,
         captainslog_assert(state->vertex_count <= vertex_buffer->Get_Vertex_Count());
 
         D3DXMATRIX mtx;
-        D3DXMatrixMultiply(
-            &mtx, (const D3DXMATRIX *)&state->sorting_state.world, (const D3DXMATRIX *)&state->sorting_state.view);
-        D3DXVECTOR3 vec;
```

Comment:
Note that `vec`, now `bounding_sphere_center`, did not have values assigned here. Thus math was wrong. Fixed.